mapeditor/graphics.cpp

```diff
@@ -48,9 +48,10 @@ void Graphics::loadPaletteAndColors()
 	for(int i = 0; i < 256; ++i)
 	{
 		QColor col;
-		col.setRed(pals[startPoint++]);
-		col.setGreen(pals[startPoint++]);
-		col.setBlue(pals[startPoint++]);
+		// Cast to unsigned char to ensure values are in the range 0-255
```

Comment:
If you want to ensure that there is no overflow then you need to use std::clamp.
Casting will not handle overflows for you